hw03_Sugargod1.py

```diff
@@ -0,0 +1,153 @@
+#################################################################################
+# Author: Sugarragchaa Tumur-Ochir
+# Username: Sugargod1
+#
+# Assignment: hw03_Sugargod1
+# Purpose: Better understanding of git and Github
+# Google Doc Link: https://docs.google.com/document/d/17NHKwPKQIHpA5JfAf-XqfP_sFllP3Gk8PNOkkDWPsdk/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+# Used Python Documentation(https://docs.python.org/3/library/turtle.html)
+#################################################################################
+
+import turtle
+
+
+def screen(wn):
+    """
+    Changes the screen color
+    """
+    wn.bgcolor("#7B68EE")
```

Comment:
This function does one thing (`wn.bgcolor("#7B68EE")`). Why not run that code directly on line 145?

Reply:
Functions that do a single line of code are not valuable, because you can just run that single line of code instead.